pep-0514.txt

```diff
@@ -13,8 +13,8 @@ Abstract
 ========
 
 This PEP defines a schema for the Python registry key to allow third-party
-installers to register their installation, and to allow applications to detect
-and correctly display all Python environments on a user's machine. No
+installers to register their installation, and to allow tools and applications
+to detect and correctly display all Python environments on a user's machine. No
 implementation changes to Python are proposed with this PEP.
 
```

Comment:
I'm still struggling to understand what (in general) tools that consume this metadata would look like. There's a lack of motivating examples here. The only examples I can think of are ```virtualenv``` and "something that lists all the available installations" (but I'm not clear what it's then going to do with the user's selection). You mention Visual Studio as another consumer of this data. It would be really useful to have a description of which data VS uses, and how it exposes it to the user. And any other motivating cases you're aware of. The other use case I can think of, unfortunately obsolete, is the ```bdist_wininst``` installer, which presented a "which Python installation do you want" dialog. But I don't recall the details of that, and it's basically obsolete now anyway. As the PEP stands, it feels like a collection of things that we think would be "nice to have", but we're not sure enough of ourselves to make them mandatory.